### DISCUSSION TITLE
[Mini-évent] Prison et joueurs

### DISCUSSION THREAD

**Discussion by ghost** [Crownicles/Crownicles#885] [Nouvelles idées]
Mini-évent permettant au joueur de découvrir un autre joueur emprisonné.

Au détour d'un chemin boueux, une petite cabane en bois cadenassée attire votre attention. C'est [Joueur] qui est emprisonné qui vous somme de l'aider à s'évader !

 ✅ l'aider à s'évader
 ❌ le laisser

Avec comme finalité :

✅ Si le joueur aide l'autre joueur à s'évader de prison : gain XP ou PO.
✅ Si le joueur échoue à le faire évader de prison : perte d'une légère poignée de PV.
❌ Si le joueur laisse l'autre joueur emprisonné, rien ne se passe.

**Comment by Pietagorh** [comment]
Et du coup ça libère l'autre joueur si on accepte et réussit ?

**Reply by ghost** [reply]
Yes ! J'aurai bien voulu que ce soit à chaque fois mais on peut faire en sorte aussi que l'évasion échoue.

**Reply by BastLast** [reply]
ca peut etre interessant ca comme ca tu peux demander à tes potes de venir sur le chemin ou tu es en prison pour te liberer !

**Reply by BastLast** [reply]
faudrait bosser dessus mais l'idée est sympa

**Reply by ghost** [reply]
Je suis contente que ça te plaise vu que j'ai eu le temps d'y réfléchir en prison... 😆
